xml/System.Net/IPAddress.xml

```diff
@@ -1686,10 +1714,12 @@
         <Parameter Name="ipString" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="ipString">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="ipString">A character span that contains an IP address in dotted-quad notation for IPv4 and in colon-hexadecimal notation for IPv6.</param>
+        <summary>Converts an IP address represented as a character span to an IPAddress instance.</summary>
```

Comment:
Do we use the language "converts" elsewhere in docs when we are parsing a string?

Reply:
@scalablecory It is used in the other [method signature](https://docs.microsoft.com/en-us/dotnet/api/system.net.ipaddress.parse?view=netcore-3.0#System_Net_IPAddress_Parse_System_String_), also it is used in [`IPEndPoint.Parse`](https://docs.microsoft.com/en-us/dotnet/api/system.net.ipendpoint.parse?view=netcore-3.0) and basically in any other type, [`int`](https://docs.microsoft.com/en-us/dotnet/api/system.int32.parse?view=netframework-4.8) in example